src/main/java/com/mraof/minestuck/data/recipe/MinestuckGristCostsProvider.java

```diff
@@ -781,6 +781,7 @@ public static void buildRecipes(Consumer<FinishedRecipe> recipeSaver)
 			if(type.getIdOrThrow().getNamespace().equals(Minestuck.MOD_ID))
 				GristCostRecipeBuilder.of(type.getCandyItem().getItem()).grist(type, 3).build(recipeSaver);
 		}
+		GristCostRecipeBuilder.of(MSItems.SOPOR_SLIME_PIE.get()).grist(URANIUM, 129).grist(RUST, 20).grist(MERCURY, 90).grist(CHALK, 15).grist(ARTIFACT, 30).build(recipeSaver);
```

Comment:
This recipe seems expensive for a food item that has both upsides and downsides. Do you have any opinions on why it should be high?